### ISSUE TITLE
Purchasing a license

### ISSUE THREAD

**Issue by Aspie96** [CGCookie/retopoflow#1068]
The README says "Purchasing a license".

Usually, with Free (as in "freedom") software, the license isn't purchased.

What is purchased is a copy of the software (or, being even more pedantic, the transmission of a copy, in case of non-phisical copies, since this isn't a DVD... but that's WAY too pedantic).

I suggest for clarity to replace "purchasing a license" with "purchasing a copy", since technically it's the latter which is purchased. I know, it's pedantic, but the former may be source of confusion about how the license works.

**Comment by vxlcoder** [comment]
Thanks for the suggestion, @Aspie96.

I agree that "Purchasing a license" is not technically correct.  I'm not sure if "copy" is the correct term to use either (I'm not a lawyer), but I do agree that it's clearer.

see commit 978caae

**Comment by Aspie96** [comment]
I realized my reply is long, so:
TL;DR: INAL, but there is much prior usage for "selling copies", so "purchasing a copy" should be perfectly ok. "purchasing the software" would be just as OK. "Purchasing a license" could, instead, lead to confusion.

Your concearn is valid and I am not a lawyer either, so this is not legal advice.

But "purchasing a copy" has prior art, which is why I suggested it: https://www.gnu.org/philosophy/selling.html

> Selling a copy of a free program is legitimate, and we encourage it.

https://www.gnu.org/licenses/gpl-faq.html#DoesTheGPLAllowMoney

> Yes, the GPL allows everyone to do this. The right to sell copies is part of the definition of free software.

The following is interesting because the questions refers to "sell a program" (as the text of the GPLv3, the very license of this project, does), but the answer refers to "selling copies" https://www.gnu.org/licenses/gpl-faq.html#GPLCommercially

> You are allowed to sell copies of the modified program commercially, but only under the terms of the GNU GPL.

The GPLv1 talks about "selling copies": ​https://www.gnu.org/licenses/old-licenses/gpl-1.0.html

In addition to this, one of the most common free software licenses, the MIT license, expressely talks about selling copies:
> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

The only possible issue with the term "copy" is that in US copyright law the term "copy" refers to a material objects, but that is specifically as used in the text of the law.

I think it is extremely unlikely the term "copy" will ever lead to any confusion, while "purchasing a license" may lead someone to believe that they are buying a permission to use the software, since that is actually what is purchased in many cases (in fact, there are many programs for which there is no charge to obtaining a copy and what you pay for is actually only the license).

It appears to me that there is just as much prior usage for "selling the software" in place of "selling a copy of the software". Although the GNU project advises against it: https://www.gnu.org/philosophy/selling.html

> The term “selling software” can be confusing too

In actuality it's also used, even on their own website, and would be just as clear, so use whichever you prefer.

**Comment by xXSeraphimXx** [comment]
So if I understand everything correctly here this means, that I can purchase it on BlenderMarket but I'm free to also just download a release here and use it for any project (i.e. commercial projects)?
